### PR TITLE
Extract shared multi-queries code

### PR DESCRIPTION
**Pull request by c-w**
No functional change.

### CODE REVIEW

File: src/clients/cassandra/CassandraConnector.js [hunk 1]
```diff
@@ -79,7 +81,23 @@ function executeQuery(query, params) { // eslint-disable-line no-unused-vars
   });
 }
 
+/**
+ * @param {Array<{query: string, params: string[]}} queries
+ * @returns {Promise.<object[]>}
+ */
+function executeQueries(queries) {
+  return new Promise((resolve, reject) => {
+    Promise.all(queries.map(query => executeQuery(query.query, query.params)))
```
Comment (Smarker) [inline src/clients/cassandra/CassandraConnector.js:90]: What about using `asyncEachLimit` instead of `Promise.all`?

Comment (c-w) [reply src/clients/cassandra/CassandraConnector.js:90]: That'll be a good future enhancement.

Comment (Smarker) [reply src/clients/cassandra/CassandraConnector.js:90]: nice ok!

**Review by Smarker — APPROVED** [review]
LGTM, and it would be nice to add `asyncEachLimit` as a future enhancement.

**Pull request merged**